<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Game Store</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QTabWidget" name="Table">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <property name="documentMode">
                <bool>false</bool>
               </property>
               <widget class="QWidget" name="tab">
                <attribute name="title">
                 <string>Game</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLineEdit" name="searchBox1"/>
                   </item>
                   <item>
                    <widget class="QPushButton" name="search1">
                     <property name="text">
                      <string>Search</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="Edit1">
                     <property name="text">
                      <string>Edit</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>Price</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="StartGame"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="text">
                      <string>~</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="EndGame"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>Start Date</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDateEdit" name="dateEdit"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_8">
                     <property name="text">
                      <string>End Date</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDateEdit" name="dateEdit_2"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QTableWidget" name="tableGame">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>4</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="sizeIncrement">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="baseSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <widget class="QPushButton" name="add1">
                     <property name="text">
                      <string>Add</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="delete1">
                     <property name="text">
                      <string>Delete</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_2">
                <attribute name="title">
                 <string>Order</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLineEdit" name="searchBox2"/>
                   </item>
                   <item>
                    <widget class="QPushButton" name="search2">
                     <property name="text">
                      <string>Search</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="Edit2">
                     <property name="text">
                      <string>Edit</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QTableWidget" name="tableOrder">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>4</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <item>
                    <widget class="QPushButton" name="add2">
                     <property name="text">
                      <string>Add</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="delete2">
                     <property name="text">
                      <string>Delete</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_3">
                <attribute name="title">
                 <string>User Info</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QLineEdit" name="searchBox3"/>
                   </item>
                   <item>
                    <widget class="QPushButton" name="search3">
                     <property name="text">
                      <string>Search</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="Edit3">
                     <property name="text">
                      <string>Edit</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QTableWidget" name="tableUser">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>4</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QPushButton" name="add3">
                     <property name="text">
                      <string>Add</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="delete3">
                     <property name="text">
                      <string>Delete</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_4">
                <attribute name="title">
                 <string>Customer</string>
                </attribute>
               </widget>
               <widget class="QWidget" name="tab_5">
                <attribute name="title">
                 <string>Store</string>
                </attribute>
                <layout class="QHBoxLayout" name="horizontalLayout_15">
                 <item>
                  <widget class="QLineEdit" name="lineEdit"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>~</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_2"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Price</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_3"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>~</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_4"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Data</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_6">
                <attribute name="title">
                 <string>Developer</string>
                </attribute>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
